### PR TITLE
M2-3342 Remove managers and respondents from global state

### PR DESCRIPTION
**Pull request by wildkiomi**

### CODE REVIEW

**Review by muhurov — APPROVED** [review]

File: src/modules/Dashboard/features/Managers/Managers.tsx [hunk 1]
```diff
@@ -19,7 +19,9 @@ import { getActions, getHeadCells } from './Managers.const';
 export const Managers = () => {
   const { t } = useTranslation('app');
   const { appletId } = useParams();
-  const dispatch = useAppDispatch();
+  const [managersData, setManagersData] = useState<{ result: Manager[]; count: number } | null>(
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/Managers/Managers.tsx:22]: Pls move '{ result: Manager[]; count: number } | null' to Managers.types.ts

**Review by alex-i-e — APPROVED** [review]

File: src/modules/Dashboard/features/Managers/Managers.tsx [hunk 2]
```diff
@@ -29,21 +31,22 @@ export const Managers = () => {
   ]);
   const rolesData = workspaces.useRolesData();
   const { ownerId } = workspaces.useData() || {};
-  const managersData = users.useManagersData();
-  const loadingStatus = users.useManagersStatus();
-  const isLoading = loadingStatus === 'loading' || loadingStatus === 'idle';
-  const { getWorkspaceManagers } = users.thunk;
+
+  const { execute: executeWorkspaceManagers, isLoading } = useAsync(
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/Managers/Managers.tsx:35]:
```suggestion
  const { execute: getWorkspaceManagers, isLoading } = useAsync(
```

File: src/modules/Dashboard/features/Respondents/Respondents.tsx [hunk 3]
```diff
@@ -38,30 +38,35 @@ import {
...
 
 export const Respondents = () => {
-  const dispatch = useAppDispatch();
   const { appletId } = useParams();
   const navigate = useNavigate();
   const { t } = useTranslation('app');
   const timeAgo = useTimeAgo();
   useBreadcrumbs();
 
+  const [respondentsData, setRespondentsData] = useState<{
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/Respondents/Respondents.tsx:47]: pls move type to Respondents.types.ts

File: src/modules/Dashboard/features/Respondents/Respondents.tsx [hunk 4]
```diff
@@ -38,30 +38,35 @@ import {
...
+  } | null>(null);
+
   const rolesData = workspaces.useRolesData();
-  const respondentsData = users.useRespondentsData();
-  const loadingStatus = users.useRespondentsStatus();
-  const isLoading = loadingStatus === 'loading' || loadingStatus === 'idle';
   const { ownerId } = workspaces.useData() || {};
-  const { getWorkspaceRespondents } = users.thunk;
+
+  const { execute: executeWorkspaceRespondents, isLoading } = useAsync(
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/Respondents/Respondents.tsx:55]:
```suggestion
  const { execute: getWorkspaceRespondents, isLoading } = useAsync(
```

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**